spring-cloud-starter-task-composedtaskrunner/src/test/java/org/springframework/cloud/task/app/composedtaskrunner/ComposedTaskStepExecutionListenerTests.java

```diff
@@ -47,12 +48,15 @@
 	private ComposedTaskStepExecutionListener taskListener;
 
 	@Before
-	public void setup() {
+	public void setup() throws NoSuchFieldException, IllegalAccessException{
 		this.taskExplorer = mock(TaskExplorer.class);
 		this.properties = new ComposedTaskProperties();
 		this.stepExecution = getStepExecution();
 		this.taskListener =
 				new ComposedTaskStepExecutionListener(this.taskExplorer);
+		Field field = this.taskListener.getClass().getDeclaredField("taskExplorer");
+		field.setAccessible(true);
+		field.set(this.taskListener, this.taskExplorer);
```

Comment:
Use `ReflectionTestUtils` for this kind of functionality: https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/util/ReflectionTestUtils.html